### ISSUE TITLE
Improvements to this plugin

### ISSUE THREAD

**Issue by ryantheriot** [kingkool68/stickyHeader#6]
- In stickyheader.jquery.js, add a setTimeout of 1 second inside the document ready to allow tables to load before using the stick header. 
- take contents of window scroll action and move it into a function that is called in the page load, as well as on the scroll action.

**Comment by kingkool68** [comment]
If you're tables load after the DOM is ready then you can try writing your own function to inject the stickyHeader script into the page after a certain amount of time. See https://api.jquery.com/jquery.getscript/ 

You can also trigger the functions associated with the scroll event like so `jQuery(window).trigger('scroll');` as a work around.

**Comment by ryantheriot** [comment]
I am using the yii framework, so It would be preferred to load the js as a content script. I just modified the code for my purposes in the both ways that i mentioned above. I was suggesting you make the same changes for friendliness to other users. 

The reason I suggested point 1 is because the if the table is not fully loaded, the header cells with have widths that do not match with the table.

The reason I suggested point 2 is because if you load into the page at the middle of a table, you would want the sticky header to show up. 

Thanks!
